package/kedro_viz/server.py

```diff
@@ -113,6 +118,31 @@ def run_viz(port=None, line=None) -> None:
     display(HTML(wrapper))
 
 
+def _load_from_file(load_file: str) -> dict:
+    global data  # pylint: disable=global-statement,invalid-name
+    data = json.loads(Path(load_file).read_text())
+    for key in ["nodes", "edges", "tags"]:
+        if key not in data:
+            raise KedroCliError(
+                "Invalid file, top level key '{}' not found.".format(key)
+            )
+    return data
+
+
+def _get_pipeline_from_context(context, pipeline_name):
+    if pipeline_name and hasattr(context, "_get_pipeline"):
+        # Kedro >=0.15.2
+        return context._get_pipeline(  # pylint: disable=protected-access
+            name=pipeline_name
+        )
+    if pipeline_name:
+        raise KedroCliError(
+            "`--pipeline` flag was provided, but the specified pipeline {} "
```

Comment:
I think this error message is a bit misleading - it fails not because the pipeline is not found, but rather because the context does not support named pipelines